docs/core/porting/tools.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Tools for porting to .NET Core
+description: Learn about some of the tools you can use to port to .NET Core
+author: cartermp
+ms.author: mairaw
+ms.date: 12/07/2018
+---
+
+# Tools to help with porting to .NET Core
+
+You may find the tools listed in this article helpful when porting:
+
+* .NET Portability Analyzer - [command line tool](https://github.com/Microsoft/dotnet-apiport/releases) or [Visual Studio Extension](https://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b), a toolchain that can generate a report of how portable your code is between .NET Framework and .NET Core, with an assembly-by-assembly breakdown of issues. For more information, see [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md).
```

Comment:
I think that the formatting could be improved for the three tools in the bulleted list to make them stand out more. For example:
- .NET Portability Analyzer
   [As a command-line tool](https://github.com/Microsoft/dotnet-apiport/releases)
   [As a Visual Studio extension](https://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b)
   ,NET Portability Analyzer is a toolchain that can generate...